modules/mod_ginger_base/templates/media/_caption.tpl

```diff
@@ -0,0 +1,8 @@
+{% if caption or id.o.actor %}
+    <figcaption>
+        {% if caption == "-" %}{% else %}
+            <p>{{ caption }}{% if m.rsc[id].o.author %} {_ By: _} <a href="{{ m.rsc[m.rsc[id].o.author[1]].page_url }}">{{ m.rsc[m.rsc[id].o.author[1]].title }}</a>{% endif %}</p>
```

Comment:
Would `id.o.author` be okay instead and in line with the style in the 1st line?

Reply:
Good one!

Reply:
I refactored this.